packages/react-i18n/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
 	"compilerOptions": {
 		"target": "ES5",
-		"lib": [ "ES2015" ],
+		"lib": [ "DOM", "ES2015" ],
```

Comment:
`DOM` lib is required for `@wordpress/compose`.

Reply:
Is that an issue with the `@types/wordpress__compose` package? The `createHigherOrderComponent` function itself doesn't depend on DOM. But `@wordpress/compose` is a somewhat random bag of various utilities, where some of them use DOM (e.g., `withGlobalEvents`). Then importing the whole `@types/wordpress__compose/index.d.ts` depends on DOM, too.

I think this is worth reporting as a Gutenberg issue. As @sgomes says, utility modules that bundle random things together can cause problems. Not only with tree shaking, but now also with types 😢

Reply:
I believe it was the `WindowEventMap` type [used here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/d129b9b777901659b1559f9355f9548496e72a2b/types/wordpress__compose/higher-order/with-global-events/index.d.ts#L4).

The type declarations do seem to have a nice tree-shakeable structure, however it does seem that types are eagerly evaluated.

> I think this is worth reporting as a Gutenberg issue.

It may be. However, type declarations are strictly third part ay this time. Gutenberg does seem open to generating type declarations, but this sort of issue seems to be somewhere between Gutenberg proper and the type declarations.